lib/appnexusapi/service.rb

```diff
@@ -46,6 +34,20 @@ def get_all(params = {})
     responses
   end
 
+  # All returns an enumerator.
+  def all(params = {})
+    Enumerator.new do |y|
```

Comment:
like this method but would rather have more expressive vars in it than `x` and `y`